src/js/node/worker_threads.ts

```diff
@@ -135,9 +135,7 @@ function fakeParentPort() {
   });
 
   Object.defineProperty(fake, "close", {
-    value() {
-      return process.exit(0);
-    },
+    value() {},
```

Comment:
Could you add a todo comment here, so we know to revisit this?

Reply:
I've added a TODO comment at the top of the `fakeParentPort()` function. Change its content if it does not fit.